<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextObjectDialog</class>
 <widget class="QDialog" name="TextObjectDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Enter your text here</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/ChartGeany.qrc">
    <normaloff>:/icons/images/icons/TIFF/edit.tiff</normaloff>:/icons/images/icons/TIFF/edit.tiff</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>480</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeComboBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>85</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="boldRadioButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </widget>
  <widget class="QLabel" name="previewLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>580</width>
     <height>42</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="fontLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="textLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="removeLbl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="removeCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>165</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="colorLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/ChartGeany.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TextObjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TextObjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
